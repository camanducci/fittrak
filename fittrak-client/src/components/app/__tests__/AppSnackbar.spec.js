import { shallowMount } from "@vue/test-utils";

import AppSnackbar from "@/components/app/AppSnackbar";

describe("AppSnackbar", () => {
  test("AppSnackbar renders", () => {
    const component = shallowMount(AppSnackbar);
    expect(component.html()).toMatchSnapshot();
  });
});
