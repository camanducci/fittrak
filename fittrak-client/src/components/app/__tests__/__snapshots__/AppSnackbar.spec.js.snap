// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppSnackbar AppSnackbar renders 1`] = `
<v-snackbar-stub timeout="7000" bottom="true" color="success">

  <v-btn-stub activeclass="v-btn--active" flat="true" tag="button" type="button">
    Close
  </v-btn-stub>
</v-snackbar-stub>
`;
